scripts/bootstrap.sh

```diff
@@ -2,19 +2,15 @@
 
 set -exo pipefail
 
-PROTOC_VER=3.13.0
-PROTOC_ZIP=protoc-${PROTOC_VER}-osx-x86_64.zip
-curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VER}/${PROTOC_ZIP}
-unzip -o ${PROTOC_ZIP} -d /usr/local bin/protoc
-unzip -o ${PROTOC_ZIP} -d /usr/local 'include/*'
-rm -f ${PROTOC_ZIP}
+source ./scripts/precheck.sh
```

Comment:
@leozc looks like you miss this new `precheck.sh` file?

Reply:
fixed.